src/main/webapp/WEB-INF/jsp/submit/enterDataForStudyEvent.jsp

```diff
@@ -380,7 +380,7 @@
                                               </c:choose>
                                             </c:when>
                                             <c:otherwise>
-                                              <span class="icon icon-file-excel red" alt="<fmt:message key="status_removed" bundle="${resword}"/>" title="<fmt:message key="status_removed" bundle="${resword}"/>"></span>
+                                              <span class="icon icon-file-excel red" alt="<fmt:message key="status_removed" bundle="${resword}"/>" title="<fmt:message key="status_removed" bundle="${resword}"/>XXX"></span>
```

Comment:
why are you adding these "X"s?

Reply:
sorry, that is part of debugging. code updated